app/src/main/java/mozilla/lockbox/autofill/FillResponseBuilder.kt

```diff
@@ -84,9 +84,13 @@ class FillResponseBuilder(
         val datasetBuilder = Dataset.Builder()
         val usernamePresentation = RemoteViews(context.packageName, mozilla.lockbox.R.layout.autofill_item)
         val passwordPresentation = RemoteViews(context.packageName, mozilla.lockbox.R.layout.autofill_item)
-        usernamePresentation.setTextViewText(mozilla.lockbox.R.id.presentationText, credential.username)
+        // sets the autofill_item text to be a username if present
+        usernamePresentation.setTextViewText(
+            mozilla.lockbox.R.id.searchLockboxText,
+            credential.username)
+        // sets the autofill_item text to be a password if present
         passwordPresentation.setTextViewText(
-            mozilla.lockbox.R.id.presentationText,
+            mozilla.lockbox.R.id.searchLockboxText,
```

Comment:
We should probably `import mozilla.lockbox.R` rather than using the fully qualified name.

I realise this isn't your error, but you're in a great position to fix it.